### DISCUSSION TITLE
Zabbix proxy

### DISCUSSION THREAD

**Discussion by Janilassila78** [zabbix-community/helm-zabbix#64] [General]
how do you config so agents can connect zabbix proxy when running it on k8s.
Should you config the agent to connect to proxys service ip and then proxy to servers service ip?

Or can anyone point me in the right direction

**Comment by aeciopires** [comment]
Hello @Janilassila78!

This is a same subject commeted in this topic: https://github.com/zabbix-community/helm-zabbix/discussions/27
